api/views/recordings_views.py

```diff
@@ -105,6 +105,38 @@ def patch(self, request, recording_id):
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
+@extend_schema(tags=["Recordings"])
+class UpdateRecordingDurationView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    @extend_schema(
+        request=RecordingDurationUpdateSerializer,
+        description="Endpoint too update the duration of the recording",
+    )
+    def patch(self, request, recording_id):
+        recording = get_object_or_404(InstructorRecordings, id=recording_id)
```

Comment:
Since the permission class will verify that the InstructorRecording object exists, I would just replace this with `InstructorRecordings.objects.get(...)`